src/PHPCR/NamespaceRegistryInterface.php

```diff
@@ -32,6 +32,11 @@
     const PREFIX_JCR = "jcr";
 
     /**
+     * A constant for the predefined namespace prefix "rep".
+     */
+    const PREFIX_REP = "rep";
```

Comment:
are you sure this is part of the specification? is that not jackrabbit that uses this for custom things?